### DISCUSSION TITLE
Filtering on QC parameter mitochondrial

### DISCUSSION THREAD

**Discussion by wudustan** [stuart-lab/signac#811] [Q&A] [answered]
Hi,

I noticed my cellranger data includes a 'mitochondrial' QC parameter. In 10x RNA-Seq that is something that is routinely used for setting filtering thresholds during QC filtering. I was wondering what people's experience was with this in 10x ATAC data? Some of my data has very high mitochondrial values.

Thanks

**Comment by timoast** [comment, marked as answer]
I usually don't filter based on mitochondrial percentage alone, but if you had cells with very high MT% you might find they have a low number of nuclear genome counts. Typically I would be less stringent in cell filtering for a first pass of the data, and then identify if there are any groups of cells identified that might be a technical artefact by looking at a combination of QC metrics: total counts, TSS enrichment score, genomic blacklist region fraction, etc.

**Reply by wudustan** [reply]
Thanks. That was my approach too, but I was just wondering in case I was missing some step.

**Comment by beginner984** [comment]
Sorry I could provide/calculate `fraction of reads in peaks` and `blacklist ratio` please?

I am able to run this formula as I can nor find this in my data

` pbmc$pct_reads_in_peaks <- pbmc$peak_region_fragments / pbmc$passed_filters * 100`
